### DISCUSSION TITLE
Add "View repository job log [here]." footer to "Dependency Dashboard" issues

### DISCUSSION THREAD

**Discussion by TWiStErRob** [renovatebot/renovate#25585] [Suggest an Idea]
## This:
![image](https://github.com/renovatebot/renovate/assets/2906988/e1d07092-bd23-4bf2-8ebf-7345a08e2239)

## Why?
When we're investigating why a certain PR is **not** opened, we would like to see the logs, but the logs are only available on PRs, but there's no PR, ...

## What did I try?
 * Going to GitHub > Settings > Applications > Renovate > Configure doesn't have this link either.
 * Going to GitHub repo > Settings > Integrations > GitHub Apps > Renovate > Configure goes to the above.
 * Opening the link (https://www.mend.io/free-developer-tools/renovate/) on top of Renovate GitHub Application goes to the Product page, not the page where we can log in.
 * In case there is an open PR in the repo, it's easy to just jump to it from the dashboard.  
   If there's no open PR however, which looks like this:
![image](https://github.com/renovatebot/renovate/assets/2906988/e072cbcc-23cc-4499-a3cf-c99b6158ecf2)
 * The only way is to jump to the GitHub repo's "Pull requests" tab, and try to find a PR that was opened by renovate.

## Workaround
Of course one could learn the URL and type it in, or bookmark it, or comment on the Dependency Dashboard issue, but when one manages many repositories, it just becomes tedious.

**Comment by rarkins** [comment]
Perhaps we could do this in the dependency dashboard header or footer, which are customizable

**Reply by TWiStErRob** [reply]
Yeah, I was looking at [`dependencyDashboardFooter`](https://docs.renovatebot.com/configuration-options/#dependencydashboardfooter), but this needs at minimum "repo org" and "repo name" placeholders (are there any placeholders for footers?), "currently running job URL" would be ideal as then there's no guessing which generated the latest update.

However even in case of [`prFooter`](https://docs.renovatebot.com/configuration-options/#prfooter) that part of the footer is not mentioned in the default. So I guess it's some internal logic adding it.

**Reply by rarkins** [reply]
Possibilities:
1. Our app can know/guess the URL in advance and inject it as part of config
2. The job ID can be logContext and we expose that as well as repo for templating
